dfinity/nns-dapp
# GIX-2016: Add Dashboard link in project detail

# Motivation

Add a link to the dashboard for easy access to more details about the SNS.

# Changes

* Add a link to the dashboard in the project detail page. Only if the project is committed.

# Tests

* Add a test for a new util to create the dashboard link.
* Add a two tests to check when the link should be present.

# Todos
